build/build-snaps.sh

```diff
@@ -22,7 +22,12 @@ then
         cp ./snapcraft-options/snapcraft-sdk.yaml ./snap/snapcraft.yaml
     fi
 
-    snapcraft
+    if [ $2 -a $2  == "arm64" ]
```

Comment:
>$2 -a $2 [](http://example.com/codeflow?start=9&length=8)

Why is the $2 -a $2 needed? Doesn't this work? `if [ $2 == "arm64" ]`

Reply:
it works, but it triggers a warning when checking this if condition if the $2 is null. So the first condition is only to remove the warning.